### PR TITLE
Fetch cities

### PR DESCRIPTION
**Pull request by quorth0n**
Depends on #1

### CODE REVIEW

**Comment by quorth0n** [comment]
Forgot my email address in the last commit, it was from a different device

File: fetch_cities.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import os
+
+URL = "https://query.wikidata.org/sparql"
+QUERY = """
+SELECT DISTINCT ?city ?cityLabel ?inception (SAMPLE(?location) AS ?location)
+WHERE
+{
+  ?city wdt:P31/wdt:P279* wd:Q515;
+        wdt:P625 ?location;
+        wdt:P1082 ?population.
```
Comment (amaury1093) [inline fetch_cities.py]: We don't need population

**Review by amaury1093 — COMMENTED** [review]
Only fetch_cities.py needs to be reviewed, correct?

**Review by amaury1093 — APPROVED** [review]
👍

**Pull request merged**